### PR TITLE
Pkg 5584 appscript 1.2.5

### PR DESCRIPTION
**Pull request by Jrice1317**
appscript 1.2.5

**Destination channel:** defaults

### Links

- [PKG-5584](https://anaconda.atlassian.net/browse/PKG-5584) 
- [Upstream repository](https://appscript.sourceforge.io/index.html)
- Relevant dependency PRs:
  - [`xlwings`](https://github.com/AnacondaRecipes/xlwings-feedstock/pull/23)

### Explanation of changes:
- Updated to last released version (no longer maintained but needed for xlwings)
- Removed patch implemented upstream in ae.c
- Addressed linter errors, including issues with use of Public Domain

[PKG-5584]: https://anaconda.atlassian.net/browse/PKG-5584?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,27 +8,25 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
Redundant

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,19 +35,24 @@ test:
...
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: http://appscript.sourceforge.net
-  license: Public Domain
+  home: https://appscript.sourceforge.io/
+  license: LicenseRef-Public-Domain
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `LicenseRef-Public-Domain-with-PSF-exception` may be more accurate, though

**Review by skupr-anaconda — APPROVED** [review]
LGTM but a non-blocking license suggestion

**Review by JamesRobertsonGames — APPROVED** [review]
Agree with @skupr-anaconda's amendments, once done feel free to merge :)

**Pull request merged**